Makefile

```diff
@@ -27,6 +28,18 @@ style:
 format:
 	black --line-length=99 $(PYFILES)
 
+## bandage:     unpack Bandage to enable assembly graphs visualizations
+bandage:
+	@if [[ "$(OS)" == "Linux" ]]; then \
+		yes A | unzip yeat/data/Bandage_Ubuntu_dynamic_v0_8_1.zip -d ~/Bandage; \
+		sudo apt install build-essential git libgl1-mesa-dev libxcb-xinerama0; \
+		sudo apt-get install libqt5gui5; \
+	elif [[ "$(OS)" == "Darwin" ]]; then \
+		yes A | unzip yeat/data/Bandage_Mac_v0_8_1.zip -d ~/Bandage; \
+	else \
+		echo "yeat does not support Bandage for: $(OS)"; \
+	fi
+
```

Comment:
> Downloaded the Bandage executables for both Linux and macOS and included them in the yeat repo.

I downloaded the Bandage executables for both Linux and macOS because of these various reasons:
1. I was unable to `conda install -c bioconda bandage`.
2. If you take a look at the last upload in anaconda, it says it was "2 years and 6 months ago".
3. When you try to conda install, you'll run into a problem where the package cannot be found in anaconda.
4. I think this stems from the problem that the author, Ryan, has said: "[He does not have] the time to give it the attention it deserves" and "don't expect a lot development or support".

To temporarily fix the problem, I added the Bandage executables from [here](https://rrwick.github.io/Bandage/), and created various commands to enable bandage in their home directory.

Reply:
As of right now, I plan on supporting both Linux and macOS. For both operating systems, if you run the entire `yeat` workflow, everything should work if the `make bandage` command was successful. 

For macOS, downloading, adding to the home directory, and using it immediately out of the box was very easy. The reason why is that the binary files needed for bandage come with the zipped `yeat/data/Bandage_Mac_v0_8_1.zip` file.
As a result, mac users should run into very little problems (hopefully).

The same cannot be said for Linux. There are a lot of challenges getting it to work on Linux.
1. the zipped file for Bandage does not contain binary files.
- As a result, you need to install various other files in order for it to work.
2. WSL2 has various other challenges that you need to install and get compared to a dedicated Linux OS
3. You need sudo. I was unable to get anything working, let alone, install necessary packages without it.

Reply:
I haven't tested thoroughly for a Linux-only system but for WSL2, I have quite a bit of experience.

In order for Bandage to run on WSL2, you'll need to do the following commands. Make sure you have sudo because you can't install anything without it.

After the Bandage executable is unzipped, you'll need to run this command at the very least:
`sudo apt-get install libqt5gui5`

This qt5 library is needed in order for bandage to run on the commandline.

In order for the qt5 GUI to open up and run, you'll need to install this:
-`conda install -c conda-forge qt`

If you are running into missing packages, try running this:
-`sudo apt install build-essential git libgl1-mesa-dev libxcb-xinerama0`
(This information was found in the installation documentation on Bandage.)

Because WSL2 is subsystem and not the main system, in order to view the GUI in the main operating system you will need to use `xserver`.

"Xserver  is an application that manages one or more graphics displays and one or more input devices (keyboard, mouse, etc.) connected to the computer.

It works as a server and can run on the local computer or on another computer on the network. Services can communicate with the X server to display graphical interfaces and receive input from the user." (from [here](https://askubuntu.com/questions/7881/what-is-the-x-server))

I'll have to write up a thing about this latter down the road. Let me know if you are interested and I'll write up a guide ASAP.

Reply:
UPDATE: Users will need to download Bandage themselves, but will keep this thread open because some of the things you need to install for Ubuntu are relevant to get Bandage to work with the already made binary file.